lib/alertwildfire_API.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2018 The Fuego Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import os
+import sys
+fuegoRoot = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.insert(0, os.path.join(fuegoRoot, 'lib'))
+sys.path.insert(0, fuegoRoot)
+import settings
+settings.fuegoRoot = fuegoRoot
+import datetime
+import requests
+import time
+import img_archive
+
+import urllib.parse as urlp
+U = urlp.urlparse('')
+U = U._replace(scheme='https', netloc='data.alertwildfire.org', path='/api/firecams/v0', params='', query='', fragment='')
+
+
+if settings.alertwildfirekey == 'do not put real key in files in open source git repo':
+    logging.warning('Please update settings.alertwildfirekey with the key provided to you')
+
+
+def get_all_camera_info():
+    headers = {'X-Api-Key': settings.alertwildfirekey}
+    Urlparts = U._replace(path = U.path+"/cameras")
```

Comment:
variable names should start with lowercase and be camelCase, so please rename to urlParts
Also, U should be treated as a constant and it would be better to copy it before replacing path.

Reply:
will rename.
U is treated as a constant as ._replace does not edit U but rather creates a new object which copies element from U and new values.
additionally i could not find an inherent copy atribute to this object but i will keep looking